modules/apps/object/object-test/src/testFunctional/tests/ObjectRelationships.testcase

```diff
@@ -1447,74 +1513,4 @@ definition {
 		ObjectPortlet.viewEntry(entry = "Entry Test");
 	}
 
-<<<<<<< HEAD
```

Comment:
Hi @JhossephG 
Can you fix SF error in just one commit, please

We can't make fixing SF errors in separate commits. For more details, you can take a look at this document: [ Test Review Guidelines](https://liferay.atlassian.net/wiki/spaces/DXPL/pages/2114356408/Test+Review+Guidelines#:~:text=Verifying%20there%20are,or%20unrelated%20commit.)